end-to-end-test/shared/specUtils_Async.js

```diff
@@ -258,6 +258,8 @@ async function goToUrlAndSetLocalStorage(url, authenticated = false) {
     const currentUrl = await browser.getUrl();
     const needToLogin =
         authenticated && (!currentUrl || !currentUrl.includes('http'));
+    // navigate to blank page first to prevent issues with url hash params
+    await browser.url('about:blank');
```

Comment:
can you see if we have an existing navigation method which does this under the hood.  i have some memory

Reply:
we had this blank page navigation in this same method name in the non async version of this spec file. but it didn't get retained when we moved this to async for some reason, so i'm just restoring this